testbench_waitress.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from waitress.channel import HTTPChannel
+from waitress.task import WSGITask
+from waitress import create_server
+
+# We need request sockets of waitress to produce "broken-stream" and "return-reset-connection"
+# error. If we are going to update waitress vesion in setup.py then code in this file may also needs to be verified and updated accordingly.
```

Comment:
Remove "If we are going ..." that is always true.  Tests need adjustment if the code / dependencies chance in incompatible way.